docs/generated/packages/react/generators/application.json

```diff
@@ -155,7 +155,7 @@
       "compiler": {
         "type": "string",
         "description": "The compiler to use.",
-        "enum": ["babel", "swc"],
+        "enum": ["swc", "babel"],
         "default": "babel"
```

Comment:
should this be set to `swc`?

Reply:
No, I realized it's not quite working with React + SWC, so I need another PR to fix that then change it. If we change it now without the fix then the setup is wrong for CSS-in-JS.